deploy/docker/seqr/Dockerfile

```diff
@@ -7,21 +7,53 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN apt-get update && apt-get install -y --no-install-recommends \
     apt-utils \
     bzip2 \
+    curl \
+    gcc \
+    libc6-dev \
+    libpq-dev \
+    nodejs \
+    npm \
+    redis-tools \
```

Comment:
not particularly important, but I doubt we need `redis-tools` for this stage of the build